modules/microshift-ki-cni-iptables-deleted.adoc

```diff
@@ -50,14 +50,15 @@ $ sudo iptables-save | grep NODEPORT
 -A OVN-KUBE-NODEPORT -p tcp -m addrtype --dst-type LOCAL -m tcp --dport 32122 -j DNAT --to-destination 10.43.17.173:80
 ----
 
-You can also confirm that a new ovnkube-master pod has been started by running the following command:
-
+. You can also confirm that a new ovnkube-master pod has been started by running the following command:
++
 [source, terminal]
 ----
 $ oc get pods -n openshift-ovn-kubernetes
 ----
-The listing of the running pods shows a new ovnkube-master pod name and age. For example:
-
+The listing of the running pods shows a new ovnkube-master pod name and age.
```

Comment:
If this is part of the same step, it needs a plus sign at the end of the terminal code block:
```suggestion
+
The listing of the running pods shows a new ovnkube-master pod name and age.
```